insta/src/components/Post.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Post = props => {
+    return (
+        <>
+            <div className='flex-column-container'>
+                <img className='thumbnail' src={props.post.thumbnailUrl} alt={'user'}></img>
```

Comment:
If you wanted to make your Post component stateful, what would be in the state of each post? How would you compose other components with a Post?